event/query_class.go

```diff
@@ -51,7 +53,7 @@ type Example struct {
 
 // NewQueryClass returns a new QueryClass for the class ID and fingerprint.
 // If example is true, the query with the greatest Query_time is saved.
```

Comment:
This pkg is fully documented (and registered with godocs official site) so please add a line to the description of this func to explain the new arg.